financialaid/api.py

```diff
@@ -32,3 +33,14 @@ def determine_auto_approval(financial_aid):
     income_threshold = COUNTRY_INCOME_THRESHOLDS.get(financial_aid.country_of_income, DEFAULT_INCOME_THRESHOLD)
     # The income_threshold == 0 is because in all cases BUT threshold == 0, it's strictly > instead of >=
     return financial_aid.income_usd > income_threshold or income_threshold == 0
+
+
+def get_no_discount_tier_program(program):
+    """
+    Takes a program_id and returns the no discount TierProgram for that Program
+    Args:
+        program (Program): the id of the Program object
+    Returns:
+        TierProgram: the no discount TierProgram program associated with the Program
+    """
+    return TierProgram.objects.get(program=program, discount_amount=0)
```

Comment:
That's fine, hopefully we have a list of the `TierProgram` objects that need to be created when a new program is set up